buildpipeline/Core-Setup-Signing-Windows-x64.json

```diff
@@ -477,7 +477,10 @@
     "GITHUB_PASSWORD": {
       "value": "PassedViaPipeBuild"
     },
-    "SIGNED_PACKAGES": {
+    "BUILD_FULL_PLATFORM_MANIFEST": {
+      "value": "true"
+    },
+    "PUBLISH_RID_AGNOSTIC_PACKAGES": {
       "value": "true"
```

Comment:
What's the difference between these vars? It seems like `BUILD_FULL_PLATFORM_MANIFEST` would be to address what @ericstj said in https://github.com/dotnet/core-setup/issues/2136#issuecomment-296316740:

>  Tests and shared framework publishing rely on having identity packages produced in all legs.

If that were the case I would expect it to be true for all legs, though.